tfchain_bridge/pkg/bridge/bridge.go

```diff
@@ -102,6 +102,32 @@ func (bridge *Bridge) Start(ctx context.Context) error {
 			if data.Err != nil {
 				return errors.Wrap(err, "failed to process events")
 			}
+
+			// Handle pending withdraws / refunds that were not yet executed
+			if data.Head-uint64(height.LastHeight) >= bridge.config.RetryInterval {
+				pendingWithdraws, err := bridge.subClient.GetPendingWithdraws()
+				if err != nil {
+					return errors.Wrap(err, "failed to get pending withdraws")
+				}
+				for _, withdraw := range *pendingWithdraws {
```

Comment:
Hmm, never range over a dereferenced pointer. Instead of pointers, return actual objects